src/emulator/apphosting/serve.ts

```diff
@@ -18,27 +24,38 @@ import { getLocalAppHostingConfiguration } from "./config";
  *    run
  *  - Dev server will respect the PORT environment variable
  */
-export async function start(): Promise<{ hostname: string; port: number }> {
+export async function start(options?: StartOptions): Promise<{ hostname: string; port: number }> {
   const hostname = DEFAULT_HOST;
   let port = DEFAULT_PORTS.apphosting;
   while (!(await availablePort(hostname, port))) {
     port += 1;
   }
 
-  serve(port);
+  serve(port, options?.customStartCommand);
 
   return { hostname, port };
 }
 
-async function serve(port: number): Promise<void> {
+async function serve(port: number, customStartCommand?: string): Promise<void> {
   const rootDir = process.cwd();
-  const packageManager = await discoverPackageManager(rootDir);
   const apphostingLocalConfig = await getLocalAppHostingConfiguration(rootDir);
-
-  await wrapSpawn(packageManager, ["run", "dev"], rootDir, {
+  const environmentVariablesToInject = {
     ...apphostingLocalConfig.environmentVariables,
     PORT: port,
-  });
+  };
+
+  if (customStartCommand) {
+    logger.logLabeled(
+      "BULLET",
+      Emulators.APPHOSTING,
+      `running custom start command: '${customStartCommand}'`,
```

Comment:
Should we print out our inferred start command to let users know what we've determined as the default?

Reply:
ya that would make sense